torso/myotorsorigid.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" ?>
+<mujoco model="FullBodyModel_MuscleActuatedLowerLimb_TorqueActuatedUpperBody">
+<!-- =================================================
+    Copyright 2020 Vikash Kumar, Vittorio Caggiano, Huawei Wang
+    Model   :: Myo Hand (MuJoCoV2.0)
+    Author  :: Vikash Kumar (vikashplus@gmail.com), Vittorio Caggiano, Huawei Wang
+    source  :: https://github.com/vikashplus
+    License :: Under Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+====================================================== -->
+
+    <include file="../scene/myosuite_scene.xml"/>
+    <include file="assets/myotorsorigid_assets.xml"/>
+    <compiler angle="radian" meshdir=".." texturedir=".."/>
+
+    <worldbody>
+        <body name="root" pos="0 0 1" euler="0 0 -1.57">
+            <include file="assets/myotorsorigid_chain.xml"/>
+            <!-- <freejoint/> -->
```

Comment:
isn't this freejoint needed for spatial movements?

Reply:
Nop